### PR TITLE
Guidance on Windows install re "Add to PATH"

### PR DESCRIPTION
**Pull request by Dangthrimble**
Updated "Installing Rust" and "After installation" to provide additional guidance to Windows users on including Rust in the Path system variable.

r? @steveklabnik

### CODE REVIEW

File: src/doc/trpl/installing-rust.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ $ sh rustup.sh
 [insecurity]: http://curlpipesh.tumblr.com
 
 If you're on Windows, please download the appropriate [installer][install-page].
+**NOTE:** By default, the Windows installer will not add Rust to the Path system
```
Comment (steveklabnik) [inline src/doc/trpl/installing-rust.md]: Shouldn't this be `%PATH%` or something?

**Comment by steveklabnik** [comment]
One small formatting nit, but otherwise, looks great :+1:

File: src/doc/trpl/installing-rust.md (hunk 1 above)
Comment (Dangthrimble) [inline src/doc/trpl/installing-rust.md]: Happy for it to be %PATH%.

----- Original Message -----

**Comment by Dangthrimble** [comment]
Steve,
On the next page, "Hello, world!", it states "Rust is an ‘expression
oriented’ language" without explaining what that means. There is a
Wikipedia page on  Expression-oriented programming language [1].
Would it be worth adding a link to that or some other page to provide
some context?
Jonathan

----- Original Message -----

**Comment by steveklabnik** [comment]
@Dangthrimble , we prefer to use our own http://doc.rust-lang.org/stable/book/glossary.html rather than link to wikipedia, as one of the main uses of rust's docs is offline, where external links don't work. we also don't control the quality of content on wikipedia. 

I would prefer just updating this PR to fix the `%PATH` thing, and then making another with that, if you don't mind :)

**Comment by Dangthrimble** [comment]
@Steveklabnik

I presume you would like me to fix %PATH%. I'll get onto it as soon as I have a spare moment.

I'm working my way through The Book (slowly!) and am happy to separately offer up changes as I work through for you to consider. Is it OK if I update the glossary as well?

**Comment by steveklabnik** [comment]
Yup! Whenever you have the chance, no rush. Same with the glossary, I would love to see it significantly expanded. :)

**Comment by Dangthrimble** [comment]
Path now updated to %PATH%.

**Comment by steveklabnik** [comment]
Thank you!

**Pull request merged**